cdap-ui/app/cdap/components/PaidBanner/index.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React from 'react';
+require('./PaidBanner.scss');
+
+export default function PaidBanner(props) {
+  if (props.expandedView) {
+    return (
+      <div className="paid-banner large">
```

Comment:
nit: You can reduce this to one return by conditionally adding text/classes. You can use `classnames` to conditionally add classes

Reply:
done